doc/flame/navigator.md

```diff
@@ -0,0 +1,63 @@
+
+```{flutter-app}
+:sources: ../flame/examples
+:page: navigator
+:show: widget code infobox
+
+This example app shows the use of the `Navigator` component to move across multiple screens within
+the game. In addition, the "pause" button stops time and applies visual effects to the content of
+the page below it.
+```
+
+
+# Navigator
+
+The **Navigator** is a component whose job is to manage navigation across multiple screens within
+the game. It is similar in spirit to Flutter's [Navigator][Flutter Navigator] class, except that it
+works with Flame components instead of Flutter widgets.
```

Comment:
Maybe an overlay should have the ability to be a page too? I think that would be quite useful.

Reply:
Sounds like an interesting idea. Of course, overlays would have limited functionality because they're always displayed on top of the Flame canvas, but it could still be useful to the user.

Reply:
I created issue #1762 for this